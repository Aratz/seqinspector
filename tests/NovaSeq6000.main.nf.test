nextflow_pipeline {

    name "Test Workflow main.nf on NovaSeq6000 data"
    script "../main.nf"
    tag "seqinspector"
    tag "PIPELINE"

    test("NovaSeq6000 data test") {

        when {
            params {
                outdir = "tests/results/NovaSeq6000"
                input  = "https://raw.githubusercontent.com/nf-core/test-datasets/seqinspector/testdata/NovaSeq6000/samplesheet.csv"
            }
        }

        then {
            assert workflow.success
        }

    }

}
