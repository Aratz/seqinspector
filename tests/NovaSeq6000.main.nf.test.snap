{
    "NovaSeq6000 data test": {
        "content": [
            "multiqc_citations.txt:md5,4c806e63a283ec1b7e78cdae3a923d4f",
            "multiqc_fastqc.txt:md5,5ba3f4f4ec6026a5f5d55418384dcd3e",
            "multiqc_general_stats.txt:md5,123cd6b64c9c15607405bcdd45a843d4",
            "multiqc_software_versions.txt:md5,7452f1f7aae2a8a4066c2ef6cd5ceb95",
            "multiqc_citations.txt:md5,4c806e63a283ec1b7e78cdae3a923d4f",
            "multiqc_fastqc.txt:md5,9b4fd8a6d6e8a9acabecd592f633472e",
            "multiqc_general_stats.txt:md5,8237b88ceb018d3cb1edcea62d10f4a2",
            "multiqc_software_versions.txt:md5,7452f1f7aae2a8a4066c2ef6cd5ceb95",
            "multiqc_citations.txt:md5,4c806e63a283ec1b7e78cdae3a923d4f",
            "multiqc_fastqc.txt:md5,9246a5b6b7b0410c79049fc3dbd08e92",
            "multiqc_general_stats.txt:md5,44328403f423c6f5ac9ee0a8a01e6725",
            "multiqc_software_versions.txt:md5,7452f1f7aae2a8a4066c2ef6cd5ceb95",
            "multiqc_citations.txt:md5,4c806e63a283ec1b7e78cdae3a923d4f",
            "multiqc_fastqc.txt:md5,84820276fae52d4d492831280ae6207c",
            "multiqc_general_stats.txt:md5,dd07799e5e4b9d389f9de49a852c3363",
            "multiqc_software_versions.txt:md5,7452f1f7aae2a8a4066c2ef6cd5ceb95",
            "multiqc_citations.txt:md5,4c806e63a283ec1b7e78cdae3a923d4f",
            "multiqc_fastqc.txt:md5,59ae05d89453da6f57010ffb6466f902",
            "multiqc_general_stats.txt:md5,e4629691992bfe639c01a84b90563334",
            "multiqc_software_versions.txt:md5,7452f1f7aae2a8a4066c2ef6cd5ceb95",
            "multiqc_citations.txt:md5,4c806e63a283ec1b7e78cdae3a923d4f",
            "multiqc_fastqc.txt:md5,5ba3f4f4ec6026a5f5d55418384dcd3e",
            "multiqc_general_stats.txt:md5,123cd6b64c9c15607405bcdd45a843d4",
            "multiqc_software_versions.txt:md5,7452f1f7aae2a8a4066c2ef6cd5ceb95"
        ],
        "timestamp": "2024-05-14T10:17:35.440827"
    }
}